src/plugins/cal-events/cal-events-fr.hbs

```diff
@@ -365,7 +365,126 @@
 				...
 			&lt;/li&gt;
 			...
-		&lt;/ol&gt;
+		&lt;/ul&gt;
+	&lt;/section&gt;
+&lt;/div&gt;</code></pre>
+			</details>
+		</section>
+	</section>
+
+	<section>
+		<h3>Exclure des éléments</h3>
+		<p>Dans cet exemple, certains éléments sont exclus de l'affichage sur le widget Calendrier en utilisant l'attribut <code>data-calevt-exclude</code>.</p>
+
+		<div id="cal-exclude"></div>
+
+		<div class="wb-calevt evt-anchor" data-calevt-src="cal-exclude" data-calevt-exclude=".exclude">
```

Comment:
as discussed, let use a static selector for now considering it is not currently supported by an current use case.

The data attribute `data-calevt-src` don't seem to be documented. What is it use case? I am not sure if that is needed when I look how it is used in the timepoke event. It seem to contain an ID, if that is the case may be named it like "src-id" | "bind-id" or let the author to provide a CSS selector as the source.

Reply:
The `data-calevt-src` is documented at the beginning of the documentation page:

"How to implement
1. Add a div element to the Web page with a class attribute containing "wb-calevt" and a unique name in a data-calevt-src attribute (e.g.: <div class="wb-calevt" data-calevt-src="unique-name"></div>).
...
3. Add a second div element to the Web page with an id attribute containing your data-calevt-src attribute's unique name from above (e.g., <div id="unique-name"></div>). This will hold the calendar interface."